### ISSUE TITLE
Fallback video problem on old browsers.

### ISSUE THREAD

**Issue by mahmutbayri** [hola/hola_player#4]
Hi,

We use hls format to serve our videos. In order to support old browser (especially not installed flash. Old aldroid web browser i.e. ) we add another video url to video as video source like below. 

```
<video id="content_video" class="video-js vjs-default-skin vjs-fill vjs-big-play-centered" controls>
    <source src="video.m3u8" type="application/x-mpegURL"/>
    <source src="video.mp4" type="video/mp4"/>
</video>
```
It works on modern browsers without any problem and old android browsers. In windows 7, IE 11 video doesn't  play with flash player.

It works -> http://www.mynet.com/tv/embed/4240365/1?try_hola
It doesn't work -> http://www.mynet.com/tv/embed/4240365/1?try_hola&try_fallback

I use browserstack for testing.

Thanks for your answer.

**Comment by akashta** [comment]
Hi,
Fixed in version 1.0.22
Thank you for reporting this issue.

**Comment by mahmutbayri** [comment]
Now, it works.

Thanks.
